docs/examples/compute/profitbricks/create_datacenter.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
Minor thing - those example files are missing Apache license headers.

Reply:
Actually, please ignore that.

It looks like none of our docs code examples contain Apache license headers right now and we explicitly mention examples are licensed under Apache 2.0 on our documentation index page.